content/docs/04.6-vm-management/15-vm-roles-permissions.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "VM User Roles and Permissions"
+metaTitle: "VM User Roles and Permissions"
+metaDescription: "Learn how to "
+icon: " "
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Overview
+
+You must configure roles and role binding before any user, including you as administrator, can access the VM Management functionality in Palette. There are two sets of roles: Cluster Roles and Palette Roles, along with the required bindings configuration.
+
+Palette provides the following four out-of-the-box Cluster roles for VM Management. The table also lists the corresponding Palette roles. 
+
+<br />
+
+| Cluster Role  | Description | Restrictions | Palette Role |
+|-----------|-------------|-----------|-----------|
+| spectro-vm-admin | Has admin privileges to manage the Kubernetes cluster, VMs, and templates.| None | - Cluster Admin or Editor<br/>- Cluster Profile Admin or Editor<br />- VM Admin
+| spectro-vm-power-user | Can perform most VM operations, but does not handle infrastructure aspects. | - Cannot manage or administer the Kubernetes cluster.<br />- Cannot manage or update VM templates. | - Cluster Viewer<br />- VM Power User |
+| spectro-vm-user | Primarily uses VMs created by others. | - Cannot launch new VMs or clone existing ones.<br />- Cannot resize or delete.<br />- Cannot migrate VMs from one node to another. | - Cluster Viewer<br />- VM User 
+| spectro-vm-viewer | A view-only role. | Cannot perform any of the operations offered to the above users.| - Cluster Viewer<br />- VM Viewer
+
+
+<br />
+
+<WarningBox>
+
+These roles are currently only relevant to access VM Management APIs. To access the Virtual Machines console, users must have permissions to access Palette clusters. These permissions can be granted through the pre-defined Cluster Admin/Editor/Viewer roles.
+
+</WarningBox>
+
+<br />
+
+You can create additional roles based on the permissions granularity that Palette offers. Palette provides the ability to specify bindings to configure granular Role-Based Access Control (RBAC) rules.
+
+<br />
+
+
+You can configure namespaces and RBAC from within a cluster or from a Palette workspace that contains a cluster group. In a cluster group all roleBindings must occur at the namespace level. For details, review the [Cluster RBAC](/clusters/cluster-management/cluster-rbac/) and [workspace RBAC](/workspace/#rolebasedaccesscontrol(rbac)) guides.  
+
+Palette leverages Regex Pattern matching so you can select multiple namespaces to apply role binding. Check out [Regex for Namespaces](/workspace/workload-features) to learn more.
+
+
```

Comment:
```suggestion
<br />
```
This adds some space so it looks less busy. 
![image](https://user-images.githubusercontent.com/29551334/226403116-787f7e59-206e-4053-bfec-21790c32cf13.png)

Reply:
done